sdk/src/instructions/event.ts

```diff
@@ -74,20 +60,18 @@ export const event = (sdk: CubikSDK) => {
         sdk.programId
       );
     },
-    getEventJoin: (pda: web3.PublicKey) => {
-      return sdk.program.account.eventJoin.fetch(pda);
-    },
 
     get: (pda: web3.PublicKey) => {
       return sdk.program.account.event.fetch(pda);
     },
-    getEventJoinPDA: (
+    
+    getEventParticipantPDA: (
```

Comment:
move pda inside participant so

so it looks like
cubiksdk.event.participant.getPDA() in the code